cdap-proto/src/main/java/co/cask/cdap/proto/ProtoTrigger.java

```diff
@@ -35,10 +37,37 @@
    * Represents all known trigger types in REST requests/responses.
    */
   public enum Type {
-    TIME,
-    PARTITION,
-    STREAM_SIZE,
-    PROGRAM_STATUS
+    TIME("time"),
```

Comment:
it will only be used in query. Don't want to make JSON incompatible with older versions